src/templates/app/_includes/authenticate_message.jsx

```diff
@@ -98,6 +128,22 @@ const FileSelector = ({
                                 );
                             })}
                         </div>
+                        { type === 'nigeria' && (
+                            <React.Fragment>
+                                <p className='font-s'>{it.L('Upload your selfie')}</p>
+                                <div className='files'>
+                                    <h3>{it.L('Your selfie')}</h3>
+                                    <div className='fields'>
+                                        <div className='gr-row form-row gr-centered'>
+                                            <div className='gr-12'>
+                                                <input id='selfie' className='file-picker' type='file' accept='.jpg, .jpeg, .gif, .png' data-type='other' data-name={it.L('Your selfie')} data-page-type='photo' />
```

Comment:
are we accepting gif 😅

Reply:
dont know but up in the same there is this and guess who did it?:)

    const accepted_formats_regex = /selfie/.test(file.passthrough.class)
            ? /^(PNG|JPG|JPEG|GIF)$/i
            : /^(PNG|JPG|JPEG|GIF|PDF)$/i;

Reply:
mm don't know should I remove? I saw that in authenticate.js there is this thats seems accepting gif also on file uploads and you are the creator seems. What should I do? thanks amami:)

      const accepted_formats_regex = /selfie/.test(file.passthrough.class)
            ? /^(PNG|JPG|JPEG|GIF)$/i
            : /^(PNG|JPG|JPEG|GIF|PDF)$/i;